website/client/components/shops/market/index.vue

```diff
@@ -23,223 +23,84 @@
-                @click.prevent.stop="togglePinned(ctx.item)"
-              )
-                span.svg-icon.inline.icon-12.color(v-html="icons.pin")
 
           keys-to-kennel(v-if='category.identifier === "special"')
 
         div.fill-height
 
-      //- @TODO: Create new InventoryDrawer component and re-use in 'inventory/stable' component.
-      drawer(
-        :title="$t('quickInventory')"
-        :errorMessage="inventoryDrawerErrorMessage(selectedDrawerItemType)"
-      )
-        div(slot="drawer-header")
-          drawer-header-tabs(
-            :tabs="drawerTabs",
-            @changedPosition="tabSelected($event)"
+      inventoryDrawer(:showEggs="true", :showPotions="true")
+        template(slot="item", slot-scope="ctx")
+          item(
+            :item="ctx.item",
+            :itemContentClass="ctx.itemClass",
+            popoverPosition="top",
+            @click="sellItem(ctx)"
           )
-            div(slot="right-item")
-              #petLikeToEatMarket.drawer-help-text(v-once)
-                | {{ $t('petLikeToEat') + ' ' }}
-                span.svg-icon.inline.icon-16(v-html="icons.information")
-              b-popover(
-                target="petLikeToEatMarket",
-                :placement="'top'",
-              )
-                .popover-content-text(v-html="$t('petLikeToEatText')", v-once)
-
-        drawer-slider(
-          v-if="hasOwnedItemsForType(selectedDrawerItemType)"
-          :items="ownedItems(selectedDrawerItemType) || []",
-          slot="drawer-slider",
-          :itemWidth=94,
-          :itemMargin=24,
-          :itemType="selectedDrawerTab"
-        )
-          template(slot="item", slot-scope="ctx")
-            item(
-              :item="ctx.item",
-              :itemContentClass="getItemClass(selectedDrawerItemType, ctx.item.key)",
-              popoverPosition="top",
-              @click="selectedItemToSell = ctx.item"
+            countBadge(
+              slot="itemBadge"
+              :show="true",
+              :count="ctx.itemCount"
             )
-              template(slot="itemBadge", slot-scope="ctx")
-                countBadge(
-                  :show="true",
-                  :count="userItems[drawerTabs[selectedDrawerTab].contentType][ctx.item.key] || 0"
-                )
-              span(slot="popoverContent")
-                h4.popover-content-title {{ getItemName(selectedDrawerItemType, ctx.item) }}
-
-      sellModal(
-        :item="selectedItemToSell",
-        :itemType="selectedDrawerItemType",
-        :itemCount="selectedItemToSell != null ? userItems[drawerTabs[selectedDrawerTab].contentType][selectedItemToSell.key] : 0",
-        :text="selectedItemToSell != null ? getItemName(selectedDrawerItemType, selectedItemToSell) : ''",
-        @change="resetItemToSell($event)"
-      )
+            h4.popover-content-title(slot="popoverContent") {{ ctx.itemName }}
+
+      sellModal
         template(slot="item", slot-scope="ctx")
```

Comment:
I thin we can remove this slot since we don't customize the sell modal "item" slot anywhere. We should be able to move all child elements to the modal's component.